### PR TITLE
Move CA articles from Istio in practice to Istio CA Certs integration

### PR DESCRIPTION
**Pull request by peterj**

### CODE REVIEW

**Comment by ZackButcher** [comment]
I'd rather have the AWS ACM setup docs after the getistio stuff, or maybe we could have it collapsed initially with a "If you don't have AWS ACM setup, start there; once it's set up, use getistio gen-ca ...". ATM looking at the page it kinda scares me, but really all the complex looking stuff is just setup.

**Comment by peterj** [comment]
I'll check if there's anything in Hugo that allows us to do the collapsible sections. 

Having the AWS ACM setup later might not make sense if they don't have the CA set up and they are trying to follow along (e.g. they won't know what the ARN or CA name will be before creating the resources).

**Comment by peterj** [comment]
Looks like we can use the details/summary construct that comes with HTML5 
Screen Shot 2021-02-10 at 9 33 00 AM

**Comment by peterj** [comment]
Fixes #138

File: content/en/istio-ca-certs-integrations/gcp-cas-integration/_index.md [hunk 1]
```diff
@@ -12,44 +12,74 @@ This approach enables the same root of trust for the root CA's workloads in GCP
...
+### Setting up CAS
+
+The first thing we need is to set up the CAS in Google Cloud Console. Log in to your Google Cloud account and follow the steps below to create a CAS instance.
+
+1. From the navigation menu, select Security → Certificate Authority Service.
+1. Click the **Create CA** button.
+1. Configure the CA type:
+    1. Select **Root CA**.
+    1. Select **365 days** for validity.
+    1. Select the **Enterprise** tier**.
```
Comment (ZackButcher) [inline content/en/istio-ca-certs-integrations/gcp-cas-integration/_index.md]:
```suggestion
    1. Select the **Enterprise** tier.
```

Did we lose a note on Enterprise tier moving the content over?

**Review by ZackButcher — APPROVED** [review]
Looks great

File: content/en/istio-ca-certs-integrations/gcp-cas-integration/_index.md [hunk 2]
```diff
@@ -12,44 +12,74 @@ This approach enables the same root of trust for the root CA's workloads in GCP
...
   secretOptions:
-    # Namespace where 'cacerts' Kubernetes secret is created on your target cluster
-    istioCANamespace: "istio-system"
-    # SecretFilePath is the file path used to store the Kubernetes Secret in yaml format
-    secretFilePath:
-    # Force flag when enabled forcefully deletes the `cacerts` secret
-    # in istioNamespace, and creates a new one.
-    force: true
+    istioCANamespace: "istio-system" # namespace where `cacerts` secrets live
+    force: true # force delete the `cacerts` secret and replace it with this new one
```
Comment (deva26) [inline content/en/istio-ca-certs-integrations/gcp-cas-integration/_index.md:75]: Just a note - need to change this flag if the other PR in getistio is released to bintray.

File: content/en/istio-ca-certs-integrations/acmpca-integration/_index.md [hunk 3]
```diff
@@ -14,44 +14,81 @@ The [`getistio gen-ca`](/getistio-cli/reference/getistio_gen-ca) command furnish
...
   secretOptions:
-    # Namespace where 'cacerts' Kubernetes secret is created on your target cluster
-    istioCANamespace: "istio-system"
-    # SecretFilePath is the file path used to store the Kubernetes Secret in yaml format
-    secretFilePath:
-    # Force flag when enabled forcefully deletes the `cacerts` secret
-    # in istioNamespace, and creates a new one.
-    force: true
+    istioCANamespace: "istio-system" # namespace where `cacerts` secrets live
+    force: true # force delete the `cacerts` secret and replace it with this new one
```
Comment (deva26) [inline content/en/istio-ca-certs-integrations/acmpca-integration/_index.md:84]: Just a note - need to change this flag if the other PR in getistio is released to bintray.

**Review by deva26 — APPROVED** [review]
Awesome, thanks @peterj

File: content/en/istio-ca-certs-integrations/gcp-cas-integration/_index.md (hunk 2 above)
Comment (peterj) [reply content/en/istio-ca-certs-integrations/gcp-cas-integration/_index.md:75]: Yep, once that changes, I'll update it here as well (I think there might be a separate issue tracking it).

File: content/en/istio-ca-certs-integrations/acmpca-integration/_index.md (hunk 3 above)
Comment (peterj) [reply content/en/istio-ca-certs-integrations/acmpca-integration/_index.md:84]: 👍🏼

**Pull request merged**